versioned_docs/version-2/platform/cloud/dcloud.md

```diff
@@ -295,6 +295,38 @@ N/A
 
 ---
 
+## Log List
+
+List log including error, warnings and info
+
+```bash
+dcloud log:list
+```
+
+**Options**
+- `--severity=<severity>` Choose the severiry of logs you wish to log (`info`, `warning` or `critical`)
```

Comment:
It seems that the option `error` is missing. Altho I do get different logs when using `critical` or `error`, it's not fully clear whats the difference to me? Also spotted a small typo in severity (severiry).

```suggestion
- `--severity=<severity>` Choose the severity of logs you wish to log (`info`, `warning`, `error` or `critical`)
```